### PR TITLE
add Azure AD login integration docs

### PR DESCRIPTION
**Pull request by viraja1**
add Azure AD login integration docs

### CODE REVIEW

File: docs/admin-tools/azure-ad-login.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+---
+title: Azure AD login integration
```
Comment (ranvijayj) [inline docs/admin-tools/azure-ad-login.md:2]: Should we use the word SSO?

**Review by ranvijayj — COMMENTED** [review]
Added comment

File: docs/admin-tools/azure-ad-login.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+Azure AD integration with the Haptik platform allows the 
+platform user to login with their existing Azure AD credentials.
+
+
+## Steps to login with Azure AD credentials
+
+1) Haptik platform user can go to the homepage and then login 
+using Microsoft option with their Azure AD credentials. 
+Once logged in, then the user will be shown a page to update Profile information. 
+Over there they can update the username and email which will be required for the Haptik Platform.
```
Comment (ranvijayj) [inline docs/admin-tools/azure-ad-login.md:14]:
```suggestion
Over there the user can update the username and email which will be required for the Haptik Platform.
```

File: docs/admin-tools/azure-ad-sso.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
+---
+title: Azure AD SSO integration
+---
+
+Azure AD integration with the Haptik platform allows the  platform user to login with their existing Azure AD
```
Comment (medhaanand) [inline docs/admin-tools/azure-ad-sso.md:5]:
```suggestion
Azure AD integration with the Haptik platform allows a user to login on the platform with their existing Azure AD 
```

**Review by medhaanand — APPROVED** [review]

**Pull request merged**